### PR TITLE
Flow typings for get-port

### PR DESCRIPTION
**Pull request by thebriando**
# ↪️ Pull Request

Adds flow typings for the `get-port` package. https://github.com/sindresorhus/get-port/blob/master/index.d.ts

## 🚨 Test instructions

Run yarn flow check and confirm there are 0 errors

### CODE REVIEW

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 8.56s | -216.00ms |
| Cached | 2.10s | +155.00ms ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 215.00ms | +31.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 257.00ms | -33.00ms 🚀 |
| dist/legacy/index.dc3d7ad2.js | 1.10kb | +0.00b | 2.29s | -214.00ms 🚀 |
| dist/legacy/index.html | 701.00b | +0.00b | 2.41s | -206.00ms 🚀 |
| dist/legacy/index.afdbda6b.css | 77.00b | +0.00b | 2.17s | -140.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 1.72s | +102.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 1.72s | +250.00ms ⚠️ |
| dist/modern/index.9c5edbbf.js | 1.10kb | +0.00b | 1.88s | +305.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 1.85s | +190.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 1.85s | +188.00ms ⚠️ |
| dist/legacy/index.afdbda6b.css | 77.00b | +0.00b | 1.85s | +189.00ms ⚠️ |
| dist/modern/index.ef3960f9.css | 77.00b | +0.00b | 1.85s | +341.00ms ⚠️ |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 26.73s | +509.00ms |
| Cached | 2.09s | -65.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/NotFound.141e0b29.js | 532.00b | +0.00b | 951.00ms | +58.00ms ⚠️ |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 108.00ms | -43.00ms 🚀 |

#### Cached Bundles

*No bundle changes detected.*

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.64m | -107.00ms |
| Cached | 5.23s | +188.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/EmojiPickerComponent.88cc2f04.js | 139.08kb | +0.00b | 12.48s | -1.04s 🚀 |
| dist/card.d65fb5b6.js | 53.76kb | +0.00b | 14.87s | -915.00ms 🚀 |
| dist/component.29a1022e.js | 30.87kb | +0.00b | 4.60s | +799.00ms ⚠️ |
| dist/esm.79a76316.js | 27.71kb | +0.00b | 13.36s | -895.00ms 🚀 |
| dist/DatePicker.a3a7f16e.js | 21.16kb | +0.00b | 10.58s | -775.00ms 🚀 |
| dist/dropzone.4a33bd89.js | 16.03kb | +0.00b | 1.52m | +35.23s ⚠️ |
| dist/ui.3fa632fc.js | 14.11kb | +0.00b | 14.87s | -918.00ms 🚀 |
| dist/workerHasher.2aeac069.js | 11.90kb | +0.00b | 13.36s | -893.00ms 🚀 |
| dist/card.ef28e700.js | 5.71kb | +0.00b | 15.72s | -886.00ms 🚀 |
| dist/EmojiPickerComponent.d78541f5.js | 3.68kb | +0.00b | 13.53s | -1.48s 🚀 |
| dist/dropzone.4ee1965e.js | 3.44kb | +0.00b | 51.50s | -2.87s 🚀 |
| dist/Modal.be4d7ece.js | 3.15kb | +0.00b | 4.59s | +802.00ms ⚠️ |
| dist/clipboard.63f58ab4.js | 2.97kb | +0.00b | 53.95s | -38.40s 🚀 |
| dist/ResourcedEmojiComponent.621cc121.js | 2.15kb | +0.00b | 14.86s | -920.00ms 🚀 |
| dist/card.766ccc7e.js | 2.15kb | +0.00b | 17.20s | -13.01s 🚀 |
| dist/feedback.85c9ab3e.js | 1.86kb | +0.00b | 10.58s | -774.00ms 🚀 |
| dist/workerHasher.8ad4130a.js | 1.75kb | +0.00b | 13.35s | -893.00ms 🚀 |
| dist/status.faf50fa4.js | 1.68kb | +0.00b | 1.52m | +1.38m ⚠️ |
| dist/heading6.4821a289.js | 1.53kb | +0.00b | 10.58s | -774.00ms 🚀 |
| dist/heading3.5de7fd77.js | 1.51kb | +0.00b | 8.47s | -1.40m 🚀 |
| dist/16.e42a1bf9.js | 1.45kb | +0.00b | 4.59s | +801.00ms ⚠️ |
| dist/heading5.78c94393.js | 1.40kb | +0.00b | 8.49s | -1.40m 🚀 |
| dist/expand.32efe718.js | 1.38kb | +0.00b | 10.58s | -771.00ms 🚀 |
| dist/16.5859a374.js | 1.34kb | +0.00b | 4.59s | +801.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.163d021e.js | 1.30kb | +0.00b | 31.36s | -23.01s 🚀 |
| dist/heading4.6f7ba126.js | 1.29kb | +0.00b | 8.47s | -1.40m 🚀 |
| dist/16.5360f030.js | 1.27kb | +0.00b | 4.59s | +801.00ms ⚠️ |
| dist/quote.e4f5d46b.js | 1.25kb | +0.00b | 1.52m | +1.38m ⚠️ |
| dist/table.f7812624.js | 1.09kb | +0.00b | 1.52m | +1.38m ⚠️ |
| dist/media-card-analytics-error-boundary.d50f5ebb.js | 1.05kb | +0.00b | 14.87s | -917.00ms 🚀 |
| dist/simpleHasher.07c456a1.js | 755.00b | +0.00b | 13.36s | -895.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.abe6ee02.js | 2.29mb | +35.00b ⚠️ | 1.16s | +135.00ms ⚠️ |
| dist/pdfRenderer.bc282f72.js | 1.11mb | +0.00b | 1.22s | +109.00ms ⚠️ |
| dist/editorView.93cc9e48.js | 587.39kb | +0.00b | 1.27s | +130.00ms ⚠️ |
| dist/popup.92f638a7.js | 169.21kb | +0.00b | 1.25s | +138.00ms ⚠️ |
| dist/EmojiPickerComponent.88cc2f04.js | 139.08kb | +0.00b | 1.22s | +128.00ms ⚠️ |
| dist/Toolbar.0ae0b492.js | 100.11kb | +0.00b | 1.28s | +133.00ms ⚠️ |
| dist/media-viewer.2cde196f.js | 72.60kb | +0.00b | 1.22s | +108.00ms ⚠️ |
| dist/card.fbdf9ace.js | 53.89kb | +0.00b | 1.20s | +111.00ms ⚠️ |
| dist/card.b7561130.js | 51.65kb | +0.00b | 1.24s | +100.00ms ⚠️ |
| dist/esm.79a76316.js | 27.71kb | +0.00b | 1.22s | +128.00ms ⚠️ |
| dist/component.b013dabf.js | 22.40kb | +0.00b | 1.04s | +67.00ms ⚠️ |
| dist/DatePicker.10cd8416.js | 21.03kb | +0.00b | 1.20s | +131.00ms ⚠️ |
| dist/smartMediaEditor.5bcd9923.js | 16.69kb | +0.00b | 1.27s | +130.00ms ⚠️ |
| dist/js.a617513d.js | 16.55kb | +0.00b | 1.04s | +68.00ms ⚠️ |
| dist/dropzone.b0d70d13.js | 15.96kb | +0.00b | 1.24s | +100.00ms ⚠️ |
| dist/ui.3fa632fc.js | 14.11kb | +0.00b | 1.20s | +111.00ms ⚠️ |
| dist/workerHasher.2aeac069.js | 11.90kb | +0.00b | 1.22s | +128.00ms ⚠️ |
| dist/component.ebf91df0.js | 6.27kb | +0.00b | 1.17s | +189.00ms ⚠️ |
| dist/card.ef28e700.js | 5.71kb | +0.00b | 1.22s | +108.00ms ⚠️ |
| dist/media-viewer.db4b7002.js | 4.01kb | +0.00b | 1.22s | +109.00ms ⚠️ |
| dist/EmojiPickerComponent.b200d273.js | 3.56kb | +0.00b | 1.20s | +111.00ms ⚠️ |
| dist/png-chunks-extract.a4983c65.js | 3.55kb | +0.00b | 1.04s | +68.00ms ⚠️ |
| dist/index.a89ddce4.css | 3.46kb | +0.00b | 1.31s | +121.00ms ⚠️ |
| dist/dropzone.4ee1965e.js | 3.44kb | +0.00b | 1.25s | +138.00ms ⚠️ |
| dist/Modal.be4d7ece.js | 3.15kb | +0.00b | 1.02s | +75.00ms ⚠️ |
| dist/clipboard.63f58ab4.js | 2.97kb | +0.00b | 1.24s | +101.00ms ⚠️ |
| dist/16.6f170271.js | 2.49kb | +0.00b | 1.16s | +139.00ms ⚠️ |
| dist/ResourcedEmojiComponent.621cc121.js | 2.15kb | +0.00b | 1.20s | +111.00ms ⚠️ |
| dist/card.766ccc7e.js | 2.15kb | +0.00b | 1.25s | +138.00ms ⚠️ |
| dist/date.70b38e9b.js | 1.96kb | +0.00b | 1.12s | +84.00ms ⚠️ |
| dist/images.07a0668d.js | 1.90kb | +0.00b | 1.13s | +97.00ms ⚠️ |
| dist/feedback.85c9ab3e.js | 1.86kb | +0.00b | 1.19s | +113.00ms ⚠️ |
| dist/16.95553deb.js | 1.86kb | +0.00b | 1.09s | +82.00ms ⚠️ |
| dist/browser.eee8b6e8.js | 1.82kb | +0.00b | 1.24s | +101.00ms ⚠️ |
| dist/16.7caa2097.js | 1.79kb | +0.00b | 1.08s | +85.00ms ⚠️ |
| dist/workerHasher.8ad4130a.js | 1.75kb | +0.00b | 1.22s | +128.00ms ⚠️ |
| dist/workerHasher.ec11e32d.js | 1.75kb | +0.00b | 1.27s | +130.00ms ⚠️ |
| dist/list-number.c28dbea9.js | 1.68kb | +0.00b | 1.14s | +84.00ms ⚠️ |
| dist/status.faf50fa4.js | 1.68kb | +0.00b | 1.16s | +98.00ms ⚠️ |
| dist/code.bae5edf4.js | 1.61kb | +0.00b | 1.12s | +85.00ms ⚠️ |
| dist/link.d81f6056.js | 1.53kb | +0.00b | 1.13s | +97.00ms ⚠️ |
| dist/heading6.4821a289.js | 1.53kb | +0.00b | 1.19s | +113.00ms ⚠️ |
| dist/heading3.5de7fd77.js | 1.51kb | +0.00b | 1.19s | +108.00ms ⚠️ |
| dist/16.81656417.js | 1.51kb | +0.00b | 1.09s | +82.00ms ⚠️ |
| dist/16.7e3cd50a.js | 1.46kb | +0.00b | 1.07s | +84.00ms ⚠️ |
| dist/16.6d662367.js | 1.46kb | +0.00b | 1.08s | +85.00ms ⚠️ |
| dist/emoji.ef33eb8c.js | 1.45kb | +0.00b | 1.13s | +97.00ms ⚠️ |
| dist/16.d73da9a6.js | 1.44kb | +0.00b | 1.11s | +87.00ms ⚠️ |
| dist/16.23416566.js | 1.40kb | +0.00b | 1.11s | +87.00ms ⚠️ |
| dist/heading5.78c94393.js | 1.40kb | +0.00b | 1.19s | +113.00ms ⚠️ |
| dist/expand.32efe718.js | 1.38kb | +0.00b | 1.19s | +113.00ms ⚠️ |
| dist/16.96477be9.js | 1.36kb | +0.00b | 1.11s | +87.00ms ⚠️ |
| dist/16.5859a374.js | 1.34kb | +0.00b | 1.02s | +75.00ms ⚠️ |
| dist/heading2.cf643822.js | 1.33kb | +0.00b | 1.19s | +108.00ms ⚠️ |
| dist/16.0349d8d1.js | 1.32kb | +0.00b | 1.11s | +87.00ms ⚠️ |
| dist/16.ac26ba20.js | 1.31kb | +0.00b | 1.12s | +84.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.163d021e.js | 1.30kb | +0.00b | 1.25s | +138.00ms ⚠️ |
| dist/mention.6e131244.js | 1.29kb | +0.00b | 1.14s | +85.00ms ⚠️ |
| dist/heading4.6f7ba126.js | 1.29kb | +0.00b | 1.19s | +108.00ms ⚠️ |
| dist/Modal.da758e16.js | 1.28kb | +0.00b | 1.04s | +68.00ms ⚠️ |
| dist/16.5360f030.js | 1.27kb | +0.00b | 1.02s | +75.00ms ⚠️ |
| dist/layout.e26f2b49.js | 1.27kb | +0.00b | 1.13s | +97.00ms ⚠️ |
| dist/16.abf1161b.js | 1.27kb | +0.00b | 1.09s | +82.00ms ⚠️ |
| dist/16.32ec918b.js | 1.26kb | +0.00b | 1.08s | +85.00ms ⚠️ |
| dist/16.67dfeae2.js | 1.26kb | +0.00b | 1.09s | +82.00ms ⚠️ |
| dist/16.ab48c5a3.js | 1.26kb | +0.00b | 1.09s | +82.00ms ⚠️ |
| dist/16.66644ac7.js | 1.26kb | +0.00b | 1.07s | +84.00ms ⚠️ |
| dist/divider.2aea4732.js | 1.25kb | +0.00b | 1.13s | +97.00ms ⚠️ |
| dist/quote.e4f5d46b.js | 1.25kb | +0.00b | 1.16s | +98.00ms ⚠️ |
| dist/component.3d077ff8.js | 1.23kb | +0.00b | 1.16s | +208.00ms ⚠️ |
| dist/action.399063c8.js | 1.23kb | +0.00b | 1.12s | +85.00ms ⚠️ |
| dist/16.600bef06.js | 1.23kb | +0.00b | 1.16s | +211.00ms ⚠️ |
| dist/decision.c50a3124.js | 1.21kb | +0.00b | 1.12s | +84.00ms ⚠️ |
| dist/panel-warning.df708655.js | 1.21kb | +0.00b | 1.16s | +98.00ms ⚠️ |
| dist/16.3e0f980c.js | 1.18kb | +0.00b | 1.11s | +87.00ms ⚠️ |
| dist/list.aaf7995b.js | 1.18kb | +0.00b | 1.14s | +85.00ms ⚠️ |
| dist/heading1.abaeff01.js | 1.18kb | +0.00b | 1.17s | +94.00ms ⚠️ |
| dist/panel-error.1a7ba35a.js | 1.11kb | +0.00b | 1.14s | +84.00ms ⚠️ |
| dist/panel.0d33c8ed.js | 1.10kb | +0.00b | 1.16s | +98.00ms ⚠️ |
| dist/table.f7812624.js | 1.09kb | +0.00b | 1.17s | +93.00ms ⚠️ |
| dist/media-picker-analytics-error-boundary.6408ae55.js | 1.05kb | +0.00b | 1.25s | +137.00ms ⚠️ |
| dist/panel-success.4e7a918e.js | 1.05kb | +0.00b | 1.16s | +98.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.d50f5ebb.js | 1.05kb | +0.00b | 1.22s | +127.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.d8e95451.js | 1.05kb | +0.00b | 1.25s | +114.00ms ⚠️ |
| dist/panel-note.c50bbb87.js | 1.05kb | +0.00b | 1.14s | +84.00ms ⚠️ |
| dist/media-viewer-analytics-error-boundary.2c4bb520.js | 995.00b | +0.00b | 1.23s | +118.00ms ⚠️ |
| dist/simpleHasher.07c456a1.js | 755.00b | +0.00b | 1.20s | +111.00ms ⚠️ |
| dist/simpleHasher.d3ebb0a0.js | 755.00b | +0.00b | 1.27s | +130.00ms ⚠️ |
| dist/index.html | 119.00b | +0.00b | 1.03s | +129.00ms ⚠️ |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/284309453572937228)**

File: packages/core/integration-tests/test/react-refresh.js [hunk 1]
```diff
@@ -183,9 +186,12 @@ async function setup(entry) {
   });
 
   subscription = await b.watch();
-  let bundleEvent = await getNextBuild(b);
+  let bundleEvent: BuildEvent = await getNextBuild(b);
+  let bundleGraph;
+  if (bundleEvent.type === 'buildSuccess') {
```
Comment (DeMoorJasper) [inline packages/core/integration-tests/test/react-refresh.js:191]: I'm sure this is probably just a hack to get flow to work, but can't this logic just be:

```js
assert.equal(bundleEvent.type, 'buildSuccess');
let bundleGraph = bundleEvent.bundleGraph;
```

**Review by DeMoorJasper — APPROVED** [review]

File: packages/core/integration-tests/test/react-refresh.js (hunk 1 above)
Comment (wbinnssmith) [reply packages/core/integration-tests/test/react-refresh.js:191]: If flow doesn't refine the value, try `invariant` aliased to `assert`.

File: packages/core/integration-tests/test/react-refresh.js [hunk 2]
```diff
@@ -205,14 +209,16 @@ async function setup(entry) {
   root = window.document.getElementById('root');
 
   let bundle = nullthrows(
-    bundleEvent.bundleGraph.getBundles().find(b => b.type === 'js'),
+    nullthrows(bundleGraph)
```
Comment (wbinnssmith) [inline packages/core/integration-tests/test/react-refresh.js:212]: is the nullthrows still needed?

File: packages/core/integration-tests/test/react-refresh.js [hunk 3]
```diff
@@ -205,14 +209,16 @@ async function setup(entry) {
...
+      .getBundles()
+      .find(b => b.type === 'js'),
   );
   let parcelRequire = Object.keys(window).find(k =>
     k.startsWith('parcelRequire'),
   );
   // ReactDOM.render
   await window[parcelRequire](
-    bundleEvent.bundleGraph.getAssetPublicId(bundle.getEntryAssets().pop()),
+    nullthrows(bundleGraph).getAssetPublicId(bundle.getEntryAssets().pop()),
```
Comment (wbinnssmith) [inline packages/core/integration-tests/test/react-refresh.js:221]: (same as above)

File: packages/core/integration-tests/test/server.js [hunk 4]
```diff
@@ -144,11 +149,12 @@ describe('server', function() {
...
     subscription = await b.watch();
-    let event = await getNextBuild(b);
-    assert.equal(event.type, 'buildSuccess');
+    let event: BuildEvent = await getNextBuild(b);
+    invariant(event.type === 'buildSuccess');
+    let bundleGraph = event.bundleGraph;
 
     let outputFile = await outputFS.readFile(
-      event.bundleGraph.getBundles()[0].filePath,
+      nullthrows(bundleGraph).getBundles()[0].filePath,
```
Comment (wbinnssmith) [inline packages/core/integration-tests/test/server.js:157]: (same as above)

**Review by wbinnssmith — APPROVED** [review]

**Pull request merged**